index.html

```diff
@@ -0,0 +1,192 @@
+<!Doctype html>
+<html>
+
+<head>
+    <meta charset="UTF-8"> 
+    <title>Moises Dobarganes Portfolio</title>
+    <link rel="stylesheet" href="./component/styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <script defer="defer" src="./component/sideScroll/sideScroll.js" ></script>
+    <div class="nav">
+        <ul class="nav__list">
+            <li>
+                <a href="#top">
+                    <img src="./img/logo.gif" /> Home
+                </a>
+            </li>
+            <li>
+                <a href="#about">About Me</a>
+            </li>
+            <li>
+                <a href="#projects">Projects</a>
+            </li>
+            <li>
+                <a href="#contact">Contact Me</a>
+            </li>
+        </ul>
+    </div>-
+    <div class="section section__bio noselect" id="bio">
+        <h1>Moises C. Dobarganes</h1>
+        <h2>Software Developer</h2>
+    </div>
+    <div class="section__divider section__divider-transition3" id="welcome"></div>
+    <div class="section section__welcome noselect">
+        <div class="section__header">Welcome <hr/></div>
+        <p>
+            Welcome to my portfolio. Scroll through this page to find a small autobiography. Halfway through there will be some images,
+            descriptions, and links to some of my projects and repositories, and towards the bottom you can find my contact
+            information. Thank you for stopping by. Feel free to browse around and be sure to let me know if there is something
+            you would like me to add.
+        </p>
+    </div>
+    <div class="section__divider" id="about"></div>
```

Comment:
"section__divider" would indicate that this element is a child of a "section" element, but it's not here nested under one.

Reply:
I think for BEM's sake, i will change "section__divider" to just "divider". Thanks again for going through my code.